assets/src/edit-story/masks/index.js

```diff
@@ -97,9 +193,175 @@ export const MASKS = [
     ratio: 1.15473441108545,
   },
   {
-    type: MaskTypes.BLOB,
-    name: __('Blob', 'web-stories'),
-    path: CLIP_PATHS[MaskTypes.BLOB],
+    type: MaskTypes.BLOB_1,
+    name: __('Blob 1', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_1],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_2,
+    name: __('Blob 2', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_2],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_3,
+    name: __('Blob 3', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_3],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_4,
+    name: __('Blob 4', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_4],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_5,
+    name: __('Blob 5', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_5],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_6,
+    name: __('Blob 6', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_6],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_7,
+    name: __('Blob 7', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_7],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_8,
+    name: __('Blob 8', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_8],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_9,
+    name: __('Blob 9', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_9],
+    iconPath: ICON_CLIP_PATHS[MaskTypes.BLOB_9],
+    ratio: 653.02 / 836.92,
+    iconRatio: 1,
+  },
+  {
+    type: MaskTypes.GRID_2,
+    name: __('Grid 2', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.GRID_2],
+    iconPath: ICON_CLIP_PATHS[MaskTypes.GRID_2],
+    ratio: 362 / 619,
+    iconRatio: 1,
+  },
+  {
+    type: MaskTypes.GRID_3,
+    name: __('Grid 3', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.GRID_3],
+    iconPath: ICON_CLIP_PATHS[MaskTypes.GRID_3],
+    ratio: 360.8 / 619,
```

Comment:
To confirm, the path itself is in the `0-1` coordinate system, right? If so, I don't see a good reason to keep this as an expression. IMHO, let's just do the math and put the value here. It's opaque from whichever way we look at it.

Reply:
So the reason we have two separate ones here are because we have no notion of strokes in clip paths. because of this when we have an item that has a 1px stroke and is 620px tall, it's stroke in the shapes panel ends up being:
```
(1px / 620px) * 36px
```
This makes the stroke invisible to nearly invisible at this size, so to combat this, we have a separate icon path that has a thicker stroke baked in, so that it's visible at an icon size.

I see us eventually getting rid of this when if/when we start handling more of the svg spec in the editor instead of just clip-paths, but this is a necessary workaround to get this working and not adding new features before beta